checkpoint_ngfw/plugin.spec.yaml

```
@@ -871,17 +847,64 @@ actions:
         example: false
         default: false
         order: 3
-      discard_other_sessions:
-        title: Discard Other Sessions
+    output:
+      success:
+        title: Success
+        description: Success
         type: boolean
-        description: Discard all other user sessions. This can fix errors when objects are locked by other sessions
         required: true
-        example: false
-        default: false
-        order: 4
+
+  check_if_address_in_group:
+    title: Check if Address in Group
+    description: Checks to see if an IPv4 or IPv6 address is in an Address Group
+    input:
+      group:
+        title: Group
+        description: Group to check. UID is not supported. Omitting this input will check all groups
+        type: string
+        required: false
+        example: InsightConnect Block List
+        order: 1
+      address:
+        title: Address
+        description: IPv4 or IPv6 address to check in the group
+        type: string
+        required: true
+        example: 198.51.100.100
+        order: 2
+    output:
+      found:
+        title: Found
+        description: Was address found in group
+        type: boolean
+        required: true
+      address_objects:
+        title: Address Objects
+        description: The names of the address objects that match or contain the address
+        type: "[]string"
+        required: true
+
+  remove_address_object_from_group:
+    title: Remove Address Object from Group
+    description: Removes an address object (host) from an address group
+    input:
+      address_object:
+        title: Address Object
+        description: The name of the address object (host) to remove
```

I think they call they these host objects? if so, change all references in the spec and regen

```suggestion
        description: The name of the address object (host object) to remove
```